templates/scenarios/js/non-sso-tab/.vscode/tasks.json

```diff
@@ -1,45 +1,84 @@
+// This file is automatically generated by Teams Toolkit.
+// The teamsfx tasks defined in this file require Teams Toolkit version >= 4.1.0.
+// See https://aka.ms/teamsfx-debug-tasks for details on how to customize each task and how to integrate with existing Teams Toolkit projects.
```

Comment:
Are these three lines of comments necessary? It seems to be not aligned with other templates.

Reply:
I'm adding these comments to all templates.
The aka links and the Teams Toolkit version will be updated later.